apps/accounts/verify.py

```diff
@@ -0,0 +1,35 @@
+# Don't import any django stuff here.
+
+import requests
+
+
+def fxa_identify(code, config=None):
+    """Get an FxA profile for an access token. If idenficiation fails return
+    an empty dict."""
+    token = get_fxa_token(code, config).get('access_token')
+    if token:
+        return get_fxa_profile(token, config)
+    else:
+        return {}
```

Comment:
this seems awkward because 1) the code will have to be defensive in always checking for an empty dict and 2) if some caller forgets to check they will get a KeyError when trying to access parameters which is harder to debug. Did you consider simply raise a `FxaProfileNotFound` exception?